### DISCUSSION TITLE
Interpreting 0 bits/s bandwidth

### DISCUSSION THREAD

**Discussion by novakboskov** [esnet/iperf#1344] [Q&A]
Server runs:
```
iperf3 -s -i 1 --daemon
```
Client runs:
```
iperf3 -c SERVER_IP -t 800 -i 1 -J
```

In the resulting JSON file, I have a streak of nearly 400 measurements that show "bytes": 0. In other words, there are nearly 400 of these and the like (consecutive):
```
{
			"streams":	[{
					"socket":	4,
					"start":	0,
					"end":	1.00006,
					"seconds":	1.00006,
					"bytes":	0,
					"bits_per_second":	0,
					"omitted":	false
				}],
			"sum":	{
				"start":	0,
				"end":	1.00006,
				"seconds":	1.00006,
				"bytes":	0,
				"bits_per_second":	0,
				"omitted":	false
			}
}
```

Everything happens over an LTE network.

How do I interpret these results?

**Comment by davidBar-On** [comment]
By default, when using TCP, iperf3 sends 128KB (1Mbits) packets.  If the network is relatively slow then it may take several seconds to send the packet.  Therefore, after the TCP buffer is full, no packet can be sent until a buffer has free space.  E.g., if your network throughput is 0.5Mbps, the on average a 128KB packet can be sent only every 2 seconds.

Can that explain what you see?  I.e. what is the average throughput of the whole test?
